python/tvm/auto_scheduler/auto_schedule.py

```diff
@@ -210,6 +253,35 @@ def auto_schedule(task, search_policy=None, tuning_options=TuningOptions()):
     if search_policy is None:
         cost_model = XGBModel()
         search_policy = SketchPolicy(task, cost_model)
+    
+    if tuning_options.check_correctness == True:
+        empty_sch, args = task.compute_dag.apply_steps_from_state(
+            task.compute_dag.get_init_state(), layout_rewrite=True)
+        cpu_func = build_module.build(
+                        empty_sch, args, target="llvm", target_host=task.target_host
```

Comment:
```suggestion
                        empty_sch, args, target="llvm", 
```
This function runs on the host machine. So we should not use `target_host`.